php/Terminus/Models/Collections/Workflows.php

```diff
@@ -171,10 +171,10 @@ public function findLatestWithLogs() {
       function($a, $b) {
         $a_finished_after_b = $a->get('finished_at') >= $b->get('finished_at');
         if ($a_finished_after_b) {
-          $cmp = 1;
+          $cmp = -1;
           return $cmp;
         } else {
-          $cmp = -1;
+          $cmp = 1;
           return $cmp;
```

Comment:
Both of these `return $cmp;` statements can be condensed into one after this if...else block.